app/models/event.rb

```diff
@@ -36,6 +38,13 @@ def publish_message
     MessageChannelService.publish_event(self)
   end
 
+  def publish_to_webhooks
+    group = self.discussion.group
```

Comment:
Publish to the webhooks associated with
- The eventable's discussion
- The eventable's discussion's group
- The eventable's discussion's group's parent group